src/chat/UnreadNotice.js

```diff
@@ -33,20 +35,29 @@ export default class UnreadNotice extends PureComponent<Props> {
 
   render() {
     const { unreadCount } = this.props;
-
-    if (unreadCount === 0) return null;
+    const visible = unreadCount !== 0;
 
     return (
-      <View style={styles.unreadContainer}>
-        <View style={styles.unreadTextWrapper}>
-          <RawLabel style={[styles.unreadText]} text={unreadToLimitedCount(unreadCount)} />
-          <Label
-            style={styles.unreadText}
-            text={unreadCount === 1 ? 'unread message' : 'unread messages'}
-          />
-        </View>
-        <MarkUnreadButton />
-      </View>
+      <AnimatedComponent
```

Comment:
The current styling is somewhat buggy. Sometimes it is narrow but visible.

Reply:
Fixed